server/services/participants.js

```diff
@@ -531,19 +548,43 @@ const getParticipants = async (
         (statusInfo) => statusInfo.status === 'hired' && !user.sites.includes(statusInfo.data.site)
       );
 
+      const hiredBySomeoneInSameOrgStatus = item.statusInfos?.find(
```

Comment:
More a nit than anything, but the function that maps the data in the return statement for this function is 50+ lines long. We should flag this for refactoring. Also all of this business logic is very hard to parse. 

I would recommend adding some more comments explaining exactly what's going on here.